lib/stream.js

```diff
@@ -52,7 +52,7 @@ try {
     // Please do not convert process.binding() to internalBinding() here.
     // This is for compatibility with older versions when loaded as
     // readable-stream.
-    Stream._isUint8Array = process.binding('util').isUint8Array;
+    Stream._isUint8Array = internalBinding('util').isUint8Array;
```

Comment:
I think we’d want to keep this one, because this code doesn’t run as part of the actual Node.js binary but is only intended for when this module is exported as readable-stream?